src/PantherTestCaseTrait.php

```diff
@@ -88,8 +88,19 @@ protected static function startWebServer(?string $webServerDir = null, string $h
         self::$baseUri = "http://$hostname:$port";
     }
 
-    protected static function createPantherClient(string $hostname = '127.0.0.1', int $port = 9000): PantherClient
+    private static function getPort(?int $port): int
```

Comment:
This function is useless, you can use a oneliner: `(int) ($port ?? $_SERVER['PANTHER_WEB_SERVER_PORT'] ?? 9000)`

Reply:
While definitely shorter, isn't a double shortened nullable ternary a bit unreadable?

Reply:
This operator has been designed for this usage, I think you can switch.
Can you also rebase please, and I'll merge.

Reply:
Ok, done